homeassistant/components/airvisual_pro/diagnostics.py

```diff
@@ -0,0 +1,36 @@
+"""Support for AirVisual Pro diagnostics."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.diagnostics import async_redact_data
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import DOMAIN
+
+CONF_MAC_ADDRESS = "mac_address"
+CONF_SERIAL_NUMBER = "serial_number"
```

Comment:
These are not used as configuration keys so we should not prefix them with `CONF_`.